app/services/AFTService.scala

```diff
@@ -36,10 +37,12 @@ class AFTService @Inject()(
 
   def fileAFTReturn(pstr: String, answers: UserAnswers)(implicit ec: ExecutionContext, hc: HeaderCarrier, request: DataRequest[_]): Future[Unit] = {
 
-    val journeyType = if (request.isAmendment) JourneyType.AFT_AMEND else JourneyType.AFT_RETURN
-
-    aftConnector.fileAFTReturn(pstr, answers, journeyType).flatMap { _ => Future.successful(())
+    val journeyType = answers.get(AFTStatusQuery) match {
```

Comment:
I think this will be clearer if use match on the 2 items with no if, i.e. (answers.get(AFTStatusQuery), request.isAmendment) match ... etc